core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/repair/OnDemandStatus.java

```diff
@@ -134,12 +139,23 @@ public OnDemandStatus(NativeConnectionProvider nativeConnectionProvider)
         myUpdateJobToFailedStatement = mySession.prepare(updateJobToFailedStatement);
     }
 
+    /**
+     * Get host ID.
+     *
+     * @return Ongoing jobs
+     */
     public UUID getHostId()
     {
         return myHostId;
     }
 
-    public Set<OngoingJob> getOngoingJobs(ReplicationState replicationState)
+    /**
+     * Create and get ongoing jobs with a replication state.
```

Comment:
Still doesn't say what it does, i was more after something like:
Reads from the table X based on A, B, C and returns in form of...

This comment applies to all methods in this file.